### PR TITLE
Feature 2 - User can play random game

### PR DESCRIPTION
**Pull request by addy-abid**
https://user-images.githubusercontent.com/85080021/141598313-3c1c56b9-da17-4351-be15-e61495138e68.mp4

### CODE REVIEW

**Comment by addy-abid** [comment]
https://user-images.githubusercontent.com/85080021/141599178-b7a5f019-9857-4234-84ce-85ef0246b4dd.mp4

**Comment by addy-abid** [comment]
Screen Shot 2021-11-15 at 10 07 18 AM
Screen Shot 2021-11-15 at 10 07 32 AM

File: js/main.js [hunk 1]
```diff
@@ -54,13 +53,90 @@ function getCharacterData(name) {
...
+      }
+      gameOptions.removeEventListener('click', clickAnswer);
+      questionCounter++;
+      if (questionCounter === 10) {
+        questionCounter = 0;
+        $modal.classList.remove('hidden');
+        $score.textContent = String(rightAnswer) + '/ 10';
+        clearInterval(timerId);
+      }
+    }
```
Comment (codingcodymiller) [inline js/main.js]: Why is this function being defined _inside_ your `load` event handler?

File: js/main.js [hunk 2]
```diff
@@ -83,3 +159,141 @@ function renderSearchResults(characters) {
...
+  $modal.classList.add('hidden');
+  for (var i = 0; i < $view.length; i++) {
+    var $dataView = event.target.getAttribute('data-view');
+    var $newView = $view[i].getAttribute('data-view');
+    if ($dataView === $newView) {
+      $view[i].className = 'container view';
+    } else {
+      $view[i].className = 'container view hidden';
+    }
+  }
```
Comment (codingcodymiller) [inline js/main.js]: Do you not have a function for switching views? 🤔

File: js/main.js [hunk 3]
```diff
@@ -83,3 +159,141 @@ function renderSearchResults(characters) {
...
+  $imgCard.appendChild($img);
+
+  var $gameQuestion = document.createElement('h2');
+  $gameQuestion.setAttribute('class', 'reem-font center-text game-card-text');
+  $gameQuestion.textContent = 'What is the name of this character?!';
+  $imgCard.appendChild($gameQuestion);
+
+  // var $hiddenColumn = document.createElement('div');
+  // $hiddenColumn.setAttribute('class', 'hidden-column-half');
+  // $imgRow.appendChild($hiddenColumn);
```
Comment (codingcodymiller) [inline js/main.js]: Please remove commented code prior to submitting for review

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**